modules/manage/partials/iceberg/query-iceberg-topics.adoc

```diff
@@ -77,7 +77,7 @@ endif::[]
 
 === Topic with schema (`value_schema_id_prefix` mode)
 
-NOTE: The steps in this section also apply to the `value_schema_latest` mode, except for step 2. The `value_schema_latest` mode doesn't require the Schema Registry wire format, so you'll use your own producer code instead of xref:reference:rpk/rpk-topic/rpk-topic-produce[`rpk topic produce`].
+NOTE: The steps in this section also apply to the `value_schema_latest` mode, except the produce step. The `value_schema_latest` mode is not compatible with the Schema Registry wire format. The xref:reference:rpk/rpk-topic/rpk-topic-produce[`rpk topic produce`] command embeds the wire format header, so you must use your own producer code with `value_schema_latest`.
```

Comment:
@r-vasquez I think it would be really great if you could produce NOT using the wire format always.  A lot of our customers don't use it, so having be a mode in your profile where you can choose how RPK produces  (and maybe consumes) would help customers a lot who don't use the wire format.